scripts/path.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
Typo in the filename here; I think you meant "patch.sh", not "path.sh", right?

Reply:
Yes exactly, that said, this file will go away.